pkgs/applications/networking/cluster/stern/default.nix

```diff
@@ -19,12 +17,11 @@ buildGoModule rec {
 
   nativeBuildInputs = [ installShellFiles ];
 
-  ldflags =
-    [ "-s" "-w" "-X github.com/stern/stern/cmd.version=${version}" ];
+  ldflags = [ "-s" "-w" "-X github.com/stern/stern/cmd.version=${version}" ];
 
-  postInstall =
-    let stern = if isCrossBuild then buildPackages.stern else "$out";
-    in
+  postInstall = let
+    stern = if stdenv.buildPlatform.canExecute stdenv.hostPlatform then buildPackages.stern else "$out";
```

Comment:
Condition should be inverted

Reply:
Should we maybe assert this? Probably not but it's bad style.

Reply:
I didn't mean invert it like that, I meant !canExecute then buildPackages